setup.py

```diff
@@ -35,12 +36,13 @@
               'bin/ddg2summary'],
 
     install_requires=[ 'biopython',
-                       'matplotlib',
-		       'numpy',
-                       'scipy',
+                       'matplotlib==3.4.3',
+		       'numpy==1.21.0',
+                       'scipy==1.6.3',
```

Comment:
can we do these changes in a separate PR, with its own issue?

Reply:
Absolutely, I can create the issue myself and I will revert the changes in the setup.py. The only question that I have is whether I should also create a new branch for the change in the setup.py and create the PR from there

Reply:
yep - new issue, connected new branch _from the `master` branch_ and associated PR